app/javascript/components/search/controls/FacetsPanel.js

```diff
@@ -1,22 +1,32 @@
 import React, { useContext } from 'react'
+
 import FacetControl from './FacetControl'
+import CombinedFacetControl from './CombinedFacetControl'
 import MoreFacetsButton from './MoreFacetsButton'
 import { SearchFacetContext } from 'providers/SearchFacetProvider'
 
+const defaultFacetIds = ['disease', 'species']
+const moreFacetIds = [
+  'sex', 'race', 'library_preparation_protocol', 'organism_age'
+]
+
 /**
  * Container for horizontal list of facet buttons, and "More Facets" button
  */
 export default function FacetsPanel() {
   const searchFacetContext = useContext(SearchFacetContext)
-
+  const defaultFacets = searchFacetContext.facets.filter(facet => defaultFacetIds.includes(facet.id))
+  const moreFacets = searchFacetContext.facets.filter(facet => moreFacetIds.includes(facet.id))
   return (
     <>
+      <CombinedFacetControl controlName="cell type" facetIds={ ['cell_type', 'cell_type__custom'] }/>
```

Comment:
Delimiting with a space strikes me as slightly unusual for non-user-facing strings (underscore or hyphen being more typical).

This isn't necessarily an issue, but it seems worth verifying -- is space-delimiting intended here?

Reply:
I definitely didn't intend to use space delimiting -- where do you see it?

Reply:
Sorry, I should've been more specific -- I was referring to `cell name`.  I mistakenly thought it was not user-facing.  No issues here.

> Maybe rename 'controlName' to 'controlDisplayName'?

A helpful measure for easily-confused readers like me :).